### DISCUSSION TITLE
chromixs sec-saxs frame similarity

### DISCUSSION THREAD

**Discussion by I-G-E** [biosaxs-com/atsas-community#1946] [Primary data processing] [answered]
Hi,

Is there a way in Chromixs to see the similarity between frames selected as the 'sample' and the similarity between frames selected as the 'buffer' - to ensure for good buffer subtraction and also selection of the best set of frames from the peak?

(I am using Chromixs from ATSAS 3.2.1).

Thank you in advance for your help.

**Comment by franke-biosaxs** [comment, marked as answer]
There should be the option to "open buffer/sample in primus". Do so and compare the frames in primus. To my knowledge, there is nothing built into chromixs.

**Reply by I-G-E** [reply]
Thank you very much for this - I was not aware of the 'open buffer/sample in primus' option so this is really helpful.
